docs/source/index.rst

```diff
@@ -0,0 +1,28 @@
+:hide-toc:
+
+=========
+Darglint2
+=========
+
+.. include:: ../../README.md
+    :parser: myst_parser.sphinx_
+    :start-after: <!-- description -->
```

Comment:
```suggestion
    :start-after: Darglint2
```
This is one way to solve the H2/H1 issue. The solution depends on how we want the top of `README.md` to look like.